docs/introduction-to-smart-contracts.rst

```diff
@@ -332,15 +332,15 @@ Storage, Memory and the Stack
 =============================
 
 Each account has a persistent memory area which is called **storage**.
-Storage is a key-value store that maps 256 bit words to 256 bit words.
+Storage is a key-value store that maps 256-bit words to 256-bit words.
```

Comment:
With the units of measurement, yes it would be with a space. For some reason, though, whenever you're talking about "xxx-bit", it's hyphenated as a convention.

Source: https://en.wikipedia.org/wiki/256-bit, https://en.wikipedia.org/wiki/8-bit, etc...